context/app/static/js/components/entity-search/SearchWrapper/utils.js

```diff
@@ -96,31 +96,72 @@ const withFacetGroup = (facetGroup) => (o) =>
 
 const createDonorFacet = withFacetGroup('Donor Metadata');
 const createDatasetFacet = withFacetGroup('Dataset Metadata');
+const createSampleFacet = withFacetGroup('Sample Metadata');
 const createAffiliationFacet = withFacetGroup('Affiliation');
 
 function mergeObjects(objects) {
   return objects.reduce((acc, curr) => ({ ...acc, ...curr }), {});
 }
 
-function getDonorMetadataFields(entityType) {
-  return [
-    createDonorFacet({
+function buildDonorFields(entityType) {
+  const tileFields = {
+    ...createDonorFacet({
       fieldName: 'sex',
       entityType,
     }),
-    createDonorFacet({
+    ...createDonorFacet({
       fieldName: 'age_value',
       entityType,
     }),
-    createDonorFacet({
+    ...createDonorFacet({
       fieldName: 'race',
       entityType,
     }),
-    createDonorFacet({
+    ...createDonorFacet({
       fieldName: 'body_mass_index_value',
       entityType,
     }),
-  ];
+  };
+  return { tableFields: tileFields, tileFields };
```

Comment:
Since we've got several places that return objects with similar structures, this might be a good place to introduce a helper class, instead of just making the object on the fly... but whatever works.